test/test_interface.cc

```diff
@@ -64,3 +64,7 @@ double TestInterface::DoubleMethod(double number) {
 const std::string TestInterface::StringMethod(const std::string& string) {
   return string;
 }
+
+void TestInterface::EnumMethod(const std::string& string) {
+  last_call_info_ = "EnumMethod(" + string + ")";
```

Comment:
The string should be IDL definition like other methods.
"void voidMethodTestEnumArg(TestEnum)";

Reply:
i fixed it.